source/column_resizer.ts

```diff
@@ -1,3 +1,15 @@
+
+export interface Rectangle {
```

Comment:
All public APIs (exported) must be documented. Some of these look redundant and can be eliminated in favor of functions.

Reply:
The idea was to model it after getBoundingClientRect() and DOMRect. I need the top, bottom, left and right.

Reply:
If you only need those then only include those.